contracts/l1/proxy/InitializableAdminUpgradeabilityProxy.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: agpl-3.0
```

Comment:
why importing all those contracts?

you could create a mock contract so that `InitializableAdminUpgradeabilityProxy` is compiled by hardhat and this is enough to use it in the tests. something like:

```solidity
pragma solidity 0.6.12;

import "@aave/protocol-v2/contracts/libraries/openzeppelin-upgradeability/InitializableAdminUpgradeabilityProxy.sol";

contract InitializableAdminUpgradeabilityProxyMock is InitializableAdminUpgradeabilityProxy {
    constructor() public InitializableAdminUpgradeabilityProxy() {}
```

Reply:
I was going to remove them anyway after the testing and rely on the imported ones